### ISSUE TITLE
[QUESTION] Llava VQA performance

### ISSUE THREAD

**Issue by benoriol** [NVIDIA/Megatron-LM#1042]
Hello,

What is the expected VQAv2 accuracy of the instruction-finetuned model?  Should it reach ~80 as reported by LLaVa1.5? After training for a few thousand steps I get under 70. Not sure if it is just a matter of training for longer or there is any other issue with the training recipe.
Also it seems the LR is 1e-6 vs the 2e-5 reported in LlaVa hyperparameters. Is this intentional? What scores did you get with this recipe?

Thanks,
Benet

Probably for @jon-barker or @trintamaki ?
